src/commonMain/kotlin/exchange.dydx.abacus/calculator/TradeInputCalculator.kt

```diff
@@ -40,9 +40,11 @@ internal class TradeInputCalculator(
     private val accountTransformer = AccountTransformer()
     private val fokDisabled = false
 
-    private val marketOrderSlippage = parser.asDecimal(0.05)!!
-    private val majorMarketSlippage = parser.asDecimal(0.05)!!
-    private val minorMarketSlippage = parser.asDecimal(0.1)!!
+    private val MARKET_ORDER_MAX_SLIPPAGE = 0.01                   // 0.05 for v3
+    private val STOP_MARKET_ORDER_SLIPPAGE_BUFFER_BTC = 0.01       // 0.05 for v3
+    private val TAKE_PROFIT_MARKET_ORDER_SLIPPAGE_BUFFER_BTC = 0.02 // 0.05 for v3
```

Comment:
instead of denoting with `_BTC` should we use `MAJOR_MARKET` (consistent with `majorMarket` flag) or `SHORT_TAIL`?